installation/scripts/release-generate-kyma-installer-artifacts.sh

```diff
@@ -15,11 +15,25 @@ set -o errexit
 
 CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 RESOURCES_DIR="${CURRENT_DIR}/../resources"
+INSTALLER_YAML_PATH="${RESOURCES_DIR}/installer.yaml"
+INSTALLER_LOCAL_CONFIG_PATH="${RESOURCES_DIR}/installer-config-local.yaml.tpl"
+INSTALLER_CLUSTER_CONFIG_PATH="${RESOURCES_DIR}/installer-config-local.yaml.tpl"
```

Comment:
```suggestion
INSTALLER_CLUSTER_CONFIG_PATH="${RESOURCES_DIR}/installer-config-cluster.yaml.tpl"
```

It definitely should be a cluster here, as this file provides different set of overrides with placeholders.